### PR TITLE
Fix the problem that multiple containers fail to start from the same qcow2 image

### PR DESCRIPTION
**Pull request by Bevisy**
升级依赖包 github.com/intel/govmm/qemu，增加qemu启动虚拟机参数

### CODE REVIEW

File: virtcontainers/qemu_amd64.go [hunk 1]
```diff
@@ -160,23 +160,25 @@ func (q *qemuAmd64) memoryTopology(memoryMb, hostMemoryMb uint64, slots uint8) g
 
 func (q *qemuAmd64) appendRootDisk(devices []govmmQemu.Device, rootDiskPath string, bootLogPath string) ([]govmmQemu.Device, error) {
 	vbp := govmmQemu.BlockDevice{
-		Driver: govmmQemu.VirtioBlockPCI,
-		ID: "virtio-disk0",
+		Driver:    govmmQemu.VirtioBlock,
+		ID:        "virtio-disk0",
```
Comment (chestack) [inline virtcontainers/qemu_amd64.go:164]: pls revert these unnecessary indent  changes

File: virtcontainers/device/config/config.go [hunk 2]
```diff
@@ -182,6 +189,12 @@ type VFIODev struct {
 
 	// DeviceID specifies device id
 	DeviceID string
+
+	// PCI Class Code
+	Class string
```
Comment (chestack) [inline virtcontainers/device/config/config.go:194]: as we talk, pls remove this parameter

Comment (Bevisy) [reply virtcontainers/device/config/config.go]: ok, already removed

**Comment by chestack** [comment]
LGTM

**Pull request merged**